android-appcoins-billing/src/main/java/com/appcoins/sdk/android_appcoins_billing/CatapultAppcoinsBilling.java

```diff
@@ -88,6 +99,10 @@ public void consumeAsync(String token, ConsumeResponseListener consumeResponseLi
   @Override public boolean isReady() {
     return billing.isReady();
   }
+
+  @Override public void OnDialogVisibleListener(boolean value) {
```

Comment:
this function should be called onDialogVisibilityChanged. This way you know it's supposed to be called every time the visibility changes and not just when the dialog is visible. Also notice the lowercase o! Java methods should always start with lowercase letter.